config/locales/en.yml

```diff
@@ -1050,10 +1050,8 @@ en:
 
       # Service attributes
       service:
-        admin_support_email: Administrative support email address
```

Comment:
I don't understand was this used for... I don't find any reference done by us in the code 🤔 it may be an active record thing 😕

Reply:
I don't think it's used anywhere. Probably it is set there in case it was used in some form, but I couldn't find reference either.